frontend/src/pages/ArtifactList.tsx

```diff
@@ -125,12 +134,12 @@ class ArtifactList extends Page<{}, PipelineListState> {
   private async getArtifactTypes():
     Promise<Map<string, MlMetadataArtifactType>> {
     try {
-      throw new Error('Not implemented yet');
-      // const response = await this.api.metadataService.listArtifactTypes();
-      // return new Map(
-      //   response.artifact_types!.map((at) => [at.id!, at])
-      // );
+      const response = await this.api.metadataService.listArtifactTypes();
+      return new Map(
+        response.artifact_types!.map((at) => [at.id!, at])
+      );
     } catch (err) {
+      // TODO: Show error since we won't be able to resolve links?
```

Comment:
Yes, I think a banner error/warning would be good for this path. We can still show the artifacts themselves because maybe that will be useful, but there should be an indication to the user that something isn't right

Reply:
Added a banner message. We won't be able to show any links in that case so I'll work to get that change added as well.